sdk/keyvault/azure-keyvault-keys/README.md

```diff
@@ -43,14 +43,13 @@ See [azure-identity][azure_identity] documentation for more information about ot
 #### Create a client
 After configuring your environment for the [DefaultAzureCredential][default_cred_ref] to use a suitable method of authentication, you can do the following to create a key client (replacing the value of `vault_url` with your vault's URL):
```

Comment:
Same comment about the variable change -- this also applies to the `azure-keyvault-secrets` README. Thank you for updating the Certificates sample, by the way!
```suggestion
After configuring your environment for the [DefaultAzureCredential][default_cred_ref] to use a suitable method of authentication, you can do the following to create a key client (replacing the value of `VAULT_URL` with your vault's URL):
```